crates/l2/contracts/deployer.rs

```diff
@@ -26,6 +30,16 @@ lazy_static::lazy_static! {
     static ref SALT: std::sync::Mutex<H256> = std::sync::Mutex::new(H256::zero());
 }
 
+type SetupResult = (
+    Address,
+    SecretKey,
+    Address,
+    Address,
+    Address,
+    EthClient,
+    PathBuf,
+);
+
```

Comment:
I've commented in https://github.com/lambdaclass/ethrex/pull/1290#discussion_r1861096826 . We can use a struct instead of a type

Reply:
I changed it in a77da2c744bd769766df3c3f5a364527fc105db3 so that it uses the same struct as in #1290